client/src/App.js

```diff
@@ -4,24 +4,24 @@ import ModalTestingPage from "./pages/ModalTestingPage";
 import Home from "./pages/Home";
 import MapSection from "./components/MapSection";
 import SliderTest from "./pages/SliderTest";
-import DataFetchingTest from "./pages/DataFetchingTest";
 
 import RhythmsPanelTestingPage from "./pages/RhythmsPanelTestingPage";
 
-
-
 const App = () => (
 	<>
-	<Routes>
-	<Route path="/" element={<Home />} />
-		<Route path="/SliderTest" element={<SliderTest />} />
-		<Route path="/ModalTestingPage" element={<ModalTestingPage />} />
-		<Route path="/" element={<Home />} />
-		<Route path="/about/this/site" element={<About />} />
-		<Route path="/ModalTestingPage" element={<ModalTestingPage />} />
-		<Route path="/RhythmsPanelTestingPage" element={<RhythmsPanelTestingPage />} />
-		<Route path="/MapSection" element={<MapSection />}></Route>
-	</Routes>
+		<Routes>
+			<Route path="/" element={<Home />} />
+			<Route path="/SliderTest" element={<SliderTest />} />
+			<Route path="/ModalTestingPage" element={<ModalTestingPage />} />
+			<Route path="/" element={<Home />} />
+			<Route path="/about/this/site" element={<About />} />
+			<Route path="/ModalTestingPage" element={<ModalTestingPage />} />
+			<Route
+				path="/RhythmsPanelTestingPage"
+				element={<RhythmsPanelTestingPage />}
+			/>
+			<Route path="/MapSection" element={<MapSection />}></Route>
+		</Routes>
```

Comment:
This just looks like a formatting change?

Reply:
Yes, this happens automatically after I uninstall and reinstall Prettier.